FeedStoreChallenge/Entities/ManagedCache+CoreDataProperties.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ManagedCache+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Istvan Private on 14.06.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+public class ManagedCache: NSManagedObject {
+	@nonobjc public class func fetchRequest() -> NSFetchRequest<ManagedCache> {
+		return NSFetchRequest<ManagedCache>(entityName: "ManagedCache")
+	}
+
+	@NSManaged public var timestamp: Date
+	@NSManaged private var feedImagesOrderedSet: NSOrderedSet
+
+	public var feedImages: [ManagedFeedImage] {
```

Comment:
Since this class should not be public, its functions and methods should be changed also.
```suggestion
	@nonobjc class func fetchRequest() -> NSFetchRequest<ManagedCache> {
		return NSFetchRequest<ManagedCache>(entityName: "ManagedCache")
	}

	@NSManaged var timestamp: Date
	@NSManaged private var feedImagesOrderedSet: NSOrderedSet

	var feedImages: [ManagedFeedImage] {
```